### ISSUE TITLE
JAWS doesn't announce aria-describedby for input element

### ISSUE THREAD

**Issue by mlhulek** [FreedomScientific/VFO-standards-support#511]
### Summary

When navigating to an input element using TAB, JAWS 2021 doesn't announce additional information which is referenced by aria-describedby. 

### Expected result

JAWS should read out the additional text which is referenced by aria-describedby.

### Actual result

JAWS doesn't read out the additional text which is referenced by aria-describedby. It used to work fine in JAWS 2019 and 2020.

### Example

https://codepen.io/mlhulek/pen/LYxPjjP

#### JAWS version and build number
JAWS 2021.2102.34

 
#### Operating System and version
MS Windows 10 Enterprise
Version 10.0.18363 Build 18363

#### Browser and version:
Chrome Version 89.0.4389.72 (Official Build) (64-Bit)

**Comment by JAWS-test** [comment]
The problem does not occur in my test environment with the same JAWS version and Chrome 89.0.4389.90. Maybe it was a temporary Chrome problem of the older version?

**Comment by mlhulek** [comment]
I just updated to Chrome Version 89.0.4389.90 (64-Bit) but the issue is still there. However, it could be a problem with Chrome since the speech output is fine in Firefox with the same JAWS version.

**Comment by JAWS-test** [comment]
That's weird. Then other people should test it, because in my test environment the output is correct. Is aria-describedby output in Chrome in your test environment for other elements? Maybe your JAWS setting is such that no description is output.

**Comment by mlhulek** [comment]
I reset the JAWS settings to get back to default. Now it is working fine. I'm sorry, it must have been some strange setting that I didn't intentionally choose. I will close this ticket. Thanks for your quick help.
